integration/experiment/power_sweep/gen_policy_recommendation.py

```diff
@@ -258,20 +258,21 @@ def policy_min_energy(plrange, enmodel, rtmodel = None, pltdp = None,
 
 
 def rootssq(ll):
-    "Return the root of the sum of the squares of the list of values."
+    """Return the root of the sum of the squares of the list of values. If a
+    non-iterable is passed in, just return it."""
     try:
         return sum([ss**2  for ss in ll])**0.5
     except TypeError:
-        pass
+        pass  # ll is not iterable
     return ll
 
 def normal_comparison(dist1, dist2):
     """Returns the probability that a sample drawn from the normal distribution
     described by dist1 (a tuple of mu and sigma) will be larger than a sample
```

Comment:
dist1 is not a tuple of mu and sigma anymore

Reply:
Done.